public/data/Jatinkumar30.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "Jatin Kumar",
+    "type": "personal",
+    "bio": "Open Source Enthusiast and Front end developer!",
+    "avatar": "https://github.com/Jatinkumar30.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Jatinkumar30",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/JatinRawal30",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Get a grip on Open Source",
+        "date": "January 2022",
+        "icon": "dollar",
```

```suggestion
        "icon": "github",
```
**It's your personal preference what icons you want to keep, but the `GitHub` icon sounds more appealing here :)**